src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -202,6 +202,38 @@ def default_api_version(self):
         ResourceType.MGMT_IOTHUB: '2020-03-01',
         ResourceType.MGMT_ARO: '2020-04-30'
     },
+    '2020-09-01-hybrid': {
```

Comment:
Please all these sepecified api versions are supported in current SDK package. If not, consider upgrading package version